src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color color = Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
You are creating a new instance of `Random` every time the method is called, which is unnecessary. Consider creating a single instance of `Random` as a private final field in the class and reusing it.